coriolisclient/v1/endpoint_storage.py

```diff
@@ -28,12 +28,12 @@ class EndpointStorageManager(base.BaseManager):
     def __init__(self, api):
         super(EndpointStorageManager, self).__init__(api)
 
-    def list(self, endpoint, environment=None):
+    def get(self, endpoint, environment=None):
         url = '/endpoints/%s/storage' % base.getid(endpoint)
 
         if environment:
             encoded_env = base64.b64encode(
                 json.dumps(environment).encode()).decode()
             url = '%s?env=%s' % (url, encoded_env)
 
-        return self._list(url, 'storage')
+        return self._get(url, 'storage')
```

Comment:
Please remember to switch back to using `client._list` (as you can see, they make the exact same HTTP request, the only added functionality of `list` being that you can pass it a key for the dict in case the return values are `namespaced`)